models/issues/issue_search.go

```diff
@@ -362,14 +362,19 @@ func applyReviewRequestedCondition(sess *xorm.Session, reviewRequestedID int64)
 		From("team_user").
 		Where(builder.Eq{"team_user.uid": reviewRequestedID})
 
+	maxReview := builder.Select("MAX(r.id)").
+		From("review as r").
+		Where(builder.In("review.type", []ReviewType{ReviewTypeApprove, ReviewTypeReject, ReviewTypeRequest}))
+
 	subQuery := builder.Select("review.issue_id").
 		From("review").
 		Where(builder.And(
-			builder.In("review.type", []ReviewType{ReviewTypeRequest, ReviewTypeReject, ReviewTypeApprove}),
+			builder.Eq{"review.type": ReviewTypeRequest},
```

Comment:
Heads up, when doing changes in queries.

Reply:
Could you point all of your PRs I can go one by one.

Reply:
Never mind. I already deleted this.